spec/javascripts/step.spec.js

```diff
@@ -7,17 +7,17 @@ describe("Step", function() {
 		Step = _Step_;
 	}));
 
-	describe("title", function() {
```

Comment:
Naming the file with capital, *Step.spec.js* instead of *step.spec.js*? To match the class's name.

Reply:
Fair enough, will do it